<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QPushButton" name="btn_mainmenu">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Return to Search</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>What would you like to do with the results?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_showresults">
         <item>
          <property name="text">
           <string>Open PDFs</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Save PDFs to zipped folder</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Open and save PDFs</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_submit">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
